adal/src/main/java/com/microsoft/aad/adal/IJWSBuilder.java

```diff
@@ -46,6 +46,6 @@
      * @param x509Certificate X509 certificate
      * @return Signed JWT
      */
-    public String generateSignedJWT(String nonce, String submitUrl, RSAPrivateKey privateKey,
+    String generateSignedJWT(String nonce, String submitUrl, RSAPrivateKey privateKey,
```

Comment:
yes, this is an interface. public does not really mean anything.  But i think i should make it protected.

---
In reply to: [65642749](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/652#discussion_r65642749) [](ancestors = 65642749)